### PR TITLE
[Merged by Bors] - chore: remove variables

### PR DESCRIPTION
**Pull request by adomani**
#17715

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Probability/Martingale/OptionalSampling.lean [hunk 1]
```diff
@@ -143,7 +143,7 @@ and is a measurable space with the Borel σ-algebra. -/
 
 variable {ι : Type*} [LinearOrder ι] [LocallyFiniteOrder ι] [OrderBot ι] [TopologicalSpace ι]
   [DiscreteTopology ι] [MeasurableSpace ι] [BorelSpace ι] [MeasurableSpace E] [BorelSpace E]
-  [SecondCountableTopology E] {ℱ : Filtration ι m} {τ σ : Ω → ι} {f : ι → Ω → E} {i n : ι}
+  [SecondCountableTopology E] {ℱ : Filtration ι m} {τ σ : Ω → ι} {f : ι → Ω → E} {n : ι}
```
Comment (grunweg) [inline Mathlib/Probability/Martingale/OptionalSampling.lean:146]: (I tend to think `i : \iota` is a more apparently name; renaming this is optional.)

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

**Comment by riccardobrasca** [comment]
Wow, this is nice! Choose the name you prefer.

bors d+

File: Mathlib/Probability/Martingale/OptionalSampling.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/Probability/Martingale/OptionalSampling.lean]: I agree that `i` is a better name, thanks!

**Comment by adomani** [comment]
Bors merge

**Pull request closed without merging**